fixtures/models/gateway/state/response_entity_details__single_account_no_assets.json

```diff
@@ -25,83 +25,9 @@
 			},
 			"metadata":
 			{
-				"total_count": 2,
+				"total_count": 0,
```

Comment:
yes, although it would be clear on a specific test for it. Will revert change and add `response_entity_details__single_resource_no_metadata.json` and its corresponding test.